ci/kind/test-e2e-kind.sh

```diff
@@ -330,7 +343,11 @@ function setup_cluster {
   fi
 
   echo "creating test bed with args $args"
-  eval "timeout 600 $TESTBED_CMD create kind $args"
+  if $flexible_ipam; then
+     eval "timeout 600 $TESTBED_CMD --flexible-ipam create kind $args"
+  else
+     eval "timeout 600 $TESTBED_CMD create kind $args"
+  fi
```

Comment:
this doesn't match the rest of the code. Why don't you add `--flexible-ipam` to `args` above?